tsconfig.app.json

```diff
@@ -4,7 +4,7 @@
     "composite": true,
     "target": "ES2020",
     "useDefineForClassFields": true,
-    "lib": ["ES2021", "DOM", "DOM.Iterable"],
+    "lib": ["ES2023", "DOM", "DOM.Iterable"],
```

Comment:
What's the motivation for this change?  And do we maybe want to use `EsNext` instead of a fixed value so we don't have to update it manually as soon as we want to use a new version? 

I'm not very familiar with this setting, however, so this could use some discussion

Reply:
We should avoid using ESNext since it could mean we find ourselves with deprecated unintended function changes. I don't remember if it was me or Ylva who changed it but it has to do with what JavaScript functions we have available, ES2023 adds some utility for Arrays for example: https://www.typescriptlang.org/tsconfig/#lib which is most likely the case here.

Reply:
Could also be that the `tsconfig.node.json` has the ´ES2023` as lib which I've changed for the same reason as above, so these should at least match each others